src/pages/AppDetails/AppDetailsPage.tsx

```diff
@@ -100,12 +152,22 @@ class AppDetails extends React.Component<RouteComponentProps<AppId>, AppDetailsS
                 <AppInfo
                   app={this.state.app}
                   namespace={this.props.match.params.namespace}
-                  onRefresh={this.fetchApp}
+                  onRefresh={this.doRefresh}
                   activeTab={this.activeTab}
                   onSelectTab={this.tabSelectHandler}
                   health={this.state.health}
                 />
               </TabPane>
+              <TabPane eventKey="traffic">
+                <TrafficDetails
+                  rateInterval={600}
```

Comment:
See the comment above about renaming this property and replacing this 600 with a constant.  I know that we'll probably be changing this to use a duration dropdown but until that time, a constant would be better to ensure the graph query and the displayed fetch duration are the same.

Reply:
Ok, I fixed the naming.